<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageTipDlg</class>
 <widget class="QDialog" name="MessageTipDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>296</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>366</width>
    <height>296</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3660</width>
    <height>2960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MessageTipDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QWidget" name="FilletWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>280</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>3500</width>
       <height>2800</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:4px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="labelIcon">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>30</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelTilte">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>110</y>
        <width>340</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>340</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>340</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18px &quot;微软雅黑&quot;;
color:#1F8BED;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelMsg">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>140</y>
        <width>340</width>
        <height>40</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>340</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>340</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font:14px &quot;微软雅黑&quot;;
color:#666666;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="BtnWidget1" native="true">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>190</y>
        <width>330</width>
        <height>80</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>330</width>
        <height>80</height>
       </size>
      </property>
      <widget class="QPushButton" name="closeButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{ background-color: #FFFFFF;border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:#333333;border:1px solid #DEDEDE}
QPushButton:hover,pressed{ background-color: #F4F4F4;border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:#333333;}</string>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="BtnWidget2" native="true">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>280</y>
        <width>330</width>
        <height>80</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>330</width>
        <height>80</height>
       </size>
      </property>
      <widget class="QPushButton" name="updateButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));
			border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
			QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40, 178, 253, 1), stop:1 rgba(64, 156, 240, 1));
		  border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}</string>
       </property>
       <property name="text">
        <string>立即升级</string>
       </property>
      </widget>
      <widget class="QPushButton" name="noUpdateButton">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{ background-color: #FFFFFF;border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:#333333;border:1px solid #DEDEDE}
QPushButton:hover,pressed{ background-color: #F4F4F4;border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:#333333;}</string>
       </property>
       <property name="text">
        <string>暂不升级</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="BtnWidget3" native="true">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>370</y>
        <width>330</width>
        <height>80</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>330</width>
        <height>80</height>
       </size>
      </property>
      <widget class="QPushButton" name="okButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));
			border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
			QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40, 178, 253, 1), stop:1 rgba(64, 156, 240, 1));
		  border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelButton">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{ background-color: #FFFFFF;border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:#333333;border:1px solid #DEDEDE}
QPushButton:hover,pressed{ background-color: #F4F4F4;border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:#333333;}</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="BtnWidget4" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>330</width>
        <height>80</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>330</width>
        <height>80</height>
       </size>
      </property>
      <widget class="QPushButton" name="bindingButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));
			border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
			QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40, 178, 253, 1), stop:1 rgba(64, 156, 240, 1));
		  border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}</string>
       </property>
       <property name="text">
        <string>登录绑定</string>
       </property>
      </widget>
      <widget class="QPushButton" name="notBindingButton">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{ background-color: #FFFFFF;border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:#333333;border:1px solid #DEDEDE}
QPushButton:hover,pressed{ background-color: #F4F4F4;border-radius:4px;font: 18px &quot;微软雅黑&quot;;color:#333333;}</string>
       </property>
       <property name="text">
        <string>暂不绑定</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="clostPushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>32</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-image: url(:/ZcloudCommon/image/browser_close_nor.png);}
QPushButton:hover,pressed{
border-image: url(:/ZcloudCommon/image/browser_close_hov.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ZcloudCommon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageTipDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageTipDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bindingButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageTipDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageTipDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageTipDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noUpdateButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageTipDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clostPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageTipDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notBindingButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageTipDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
